requirements.txt

```diff
@@ -11,12 +11,16 @@ Pillow                  ==8.0.1       # HPND
 ffmpeg-python           ==0.2.0       # GNU-GPL
 simpleaudio             ==1.0.4       # MIT
 
-# Dependencies needed for converting Pytorch models to CoreML
-tensorflow              ==1.14.0
-keras                   ==2.2.4
-coremltools             ==3.3
-h5py                    ==2.10.0
+# Dependencies needed for converting Pytorch models to tensorflow Lite
+tensorflow              ==2.3.1
+Keras                   ==2.4.3
 
 # unit tests
 pycodestyle             ==2.5.0          # MIT
-pytest                  ==5.4.3          # MIT
\ No newline at end of file
+pytest                  ==5.4.3          # MIT
+
+# Dependencies needed for converting Pytorch models to CoreML
+#tensorflow              ==1.14.0
+#keras                   ==2.2.4
+#coremltools             ==3.3
+#h5py                    ==2.10.0
```

Comment:
Commented out? Are these lines leftover from development and no longer needed?

Reply:
only needed if you want to a conversion to coreml. (explained in the comment)
conversion in tensorflow lite or coreml have different dependencies versions

Reply:
Ah, got it. Not very easy to handle for users, though, I think (and also not testable). How about we move these dependencies to a different requirements file that can be installed in addition to the normal requirements? For example, call it `requirements-coreml.txt`?